bundlewrap/items/postgres_roles.py

```diff
@@ -17,14 +18,13 @@ def delete_role(node, role):
 
 
 def fix_role(node, role, attrs, create=False):
-    password = " PASSWORD '{}'".format(attrs['password_hash'])
-    sql = "{operation} ROLE \\\"{role}\\\" WITH LOGIN {superuser}SUPERUSER{password}".format(
-        operation="CREATE" if create else "ALTER",
-        password="" if attrs['password_hash'] is None else password,
-        role=role,
-        superuser="" if attrs['superuser'] is True else "NO",
-    )
-    node.run(f"psql -nqw -c \"{sql}\"", user="postgres")
+    if create:
+        superuser_sql = "SUPERUSER" if attrs['superuser'] else "NOSUPERUSER"
```

Comment:
I’m afraid this is a breaking change. The superuser property no longer gets updated in the database.

Suppose there’s an existing user with the item attribute `superuser = True`. When changing this attribute to `False`, it must be changed in the DB as well. That’s not the case currently:

```
? foo-test  ╭─ postgres_role:testpho
? foo-test  │
? foo-test  │  superuser  yes → no
? foo-test  │
? foo-test  ╰─ Fix postgres_role:testpho? [Y/n] y
? foo-test
✘ foo-test  postgresql  postgres_role:testpho  failed
│ superuser  yes → no
╵
```